multi-node/aws/README.md

```diff
@@ -5,24 +5,86 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ## Development
 
+### Download pre-built binary
+
+```sh
+wget https://<binary-url>
+# check checksum
+chmod +x ./kube-aws
+sudo mv kube-aws /usr/bin/
```

Comment:
nit: I won't block on this, but I dislike that our instructions would require sudo for this tool. I think this could be mitigated by allowing and `--output` flag or something of the sort, so we don't rely on running the tool in the directory we wish to act on.

Or another option would be that our instructions say something like `alias kube-aws='/path/to/kube-aws'`.

Or if left as-is, this should probably be /usr/local/bin